packages/viewer/src/interactions/baseInteractionHandler.ts

```diff
@@ -195,6 +200,15 @@ export abstract class BaseInteractionHandler implements InteractionHandler {
   }
 
   protected drag(event: BaseEvent): void {
+    if (this.keyboardControls && event.altKey && event.shiftKey) {
+      this.currentInteraction = this.twistInteraction;
+    } else {
+      this.currentInteraction = undefined;
+    }
+    this.draggingInteraction =
+      this.currentInteraction ||
+      this.draggingInteraction ||
+      this.primaryInteraction;
```

Comment:
Just curious, any reason this couldn't continue to live in the `twistInteractionHandler`? Seems like either way is probably fine right now though if this resolves the bug since https://vertexvis.atlassian.net/browse/SDK-1515 will hopefully address some of the challenges around these interactions

Reply:
Chatted on zoom about this. Going to add some description that encapsulates the conversation around some of these things.